src/sensors/distance/ultrasound/i2c/SRF08.cpp

```diff
@@ -46,16 +46,14 @@ unsigned int SRF08::getDistance()
     mRuntime.i2cEndTransmission();
 
     mRuntime.i2cRequestFrom(mAddress, kNumberOfBytesToRequest);
-    if (mRuntime.i2cAvailable() == 0)
+    if (!mRuntime.i2cAvailable())
     {
         return static_cast<unsigned int>(-1); // Return a large error-like value
     }
     auto high = mRuntime.i2cRead();
     auto low  = mRuntime.i2cRead();
 
-    static constexpr auto kBitsInByte = 8;
-
-    return static_cast<uint16_t>((high << kBitsInByte) + low);
+    return static_cast<uint16_t>((high << 8) + low);
```

Comment:
:warning: **readability-magic-numbers** :warning:
8 is a magic number; consider replacing it with a named constant